risc0/zkvm/src/sha.rs

```diff
@@ -61,6 +61,19 @@ cfg_if::cfg_if! {
     }
 }
 
+/// Defines a collision resistant hash for the typed and structured data.
+pub trait Digestable {
```

Comment:
Would it be desirable to change from `Digestable` to `Digestible`?

Reply:
Yes, haha 😅  Sorry, I barely passed spelling in the third grade 😆

Reply:
Fixed this in 0e5f89aec525422ec605d83e730925f120f575f5